### ISSUE TITLE
Cannot add yabai to accessibility

### ISSUE THREAD

**Issue by Ciacconas** [koekeishiya/yabai#92]
I followed the wiki installation guide and when I'm adding yabai to system accessibility, the yabai binary is not highlighted (can't not be chosen) like figure below. (I also found the linked yabai binary, which is also gray and cannot be added)
![Screen Shot 2019-07-05 at 3 03 17 PM](https://user-images.githubusercontent.com/31930221/60724424-b083f880-9f36-11e9-8d27-3d3fa9548919.png)
My system is 10.13.6 (17G7024)
Is there anything I can do to add yabai to the accessibility?

**Comment by dominiklohmann** [comment]
If you cannot add it in there for 10.13, try running yabai once and then immediately quit it (`brew services start yabai` and `brew services stop yabai`). You will get spammed with accessibility prompts, which will add yabai as unchecked to the list of apps asking for these permissions. I really don't see why this would not work, though.

**Comment by Ciacconas** [comment]
> If you cannot add it in there for 10.13, try running yabai once and then immediately quit it (`brew services start yabai` and `brew services stop yabai`). You will get spammed with accessibility prompts, which will add yabai as unchecked to the list of apps asking for these permissions. I really don't see why this would not work, though.

It worked, thank you very much. I should have known that running yabai for once would invoke the prompts. It's been a long time since I use mac last time. Back for yabai! 👍
